### DISCUSSION TITLE
Lesson 7 AggregatorV3Interface.sol problem

### DISCUSSION THREAD

**Discussion by Crocosaurus96** [Cyfrin/foundry-full-course-cu#1457] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/106905410/aaa64148-4f9e-4815-b68a-511df5ea925d)
Hello!

I don't know what's wrong here. The AggregatorV3Interface.sol is there but I get this error that it doesn't exist.

**Comment by EngrPips** [comment]
Hello @Crocosaurus96 , your path says `/home/useradd/foundry-fund-me-f23/lib/chainlink-brownie-contracts/contracts/src/v0.8/shared/interface/AggregatorV3Interface.sol` , But in the screenshot that you shared which shows your project directory tree i can not see any folder named `shared` so where do that `shared` in the path comes from ?

**Reply by Crocosaurus96** [reply]
@EngrPips you are correct.
what I did is I run this command "forge install smartcontractkit/chainlink-brownie-contracts@0.8.0 --no-commit" and then I tried to "forge test". Do I need to change Aggregatorv3Interface.sol from the current directory to /home/useradd/foundry-fund-me-f23/lib/chainlink-brownie-contracts/contracts/src/v0.8/shared/interface/AggregatorV3Interface.sol?

**Comment by EngrPips** [comment]
@Crocosaurus96  Your file need to be in the path you provided for the contract to get it, just remove the `shared` from the path i would say and it should work fine
